tests/integration_tests/README.md

```diff
@@ -0,0 +1,18 @@
+# Integration tests
+
+
+## Requirements
+
+- Node
+- Go
+- Dep
+- Make
+- DD_API_KEY
+
+## Running
+
+```bash
+DD_API_KEY=<API_KEY> aws-vault exec sandbox-account-admin -- ./run_integration_tests.sh
```

Comment:
This is perhaps overly cautious, but is it wise to expose the names of our internal roles in an open source library?

Reply:
Great question. I followed integration tests from other libraries such as https://github.com/DataDog/datadog-lambda-python/blob/master/scripts/run_integration_tests.sh#L5 and https://github.com/DataDog/datadog-lambda-js/blob/master/scripts/run_integration_tests.sh#L5.

Happy to reassess with the pertinent teams.